src/TourCMS.php

```diff
@@ -195,6 +195,21 @@ public function list_channels() {
 		return($this->request('/p/channels/list.xml'));
 	}
 
+<<<<<<< Updated upstream
+=======
+	public function channel_upload_logo_get_url($channel)
+	{
+		$url = "/c/channel/logo/upload/url.xml";
+		return($this->request($url, $channel, self::HTTP_VERB_GET));
+	}
+
+	public function channel_upload_logo_process($channel, $upload_info)
+	{
+		$url = "/c/channel/logo/upload/process.xml";
+		return($this->request($url, $channel, self::HTTP_VERB_POST, $upload_info));
+	}
+
+>>>>>>> Stashed changes
```

Comment:
This changes would break the wrapper. Please use the latest release and check only the new changes are added to the PR